drivers/filesystems/ext2/inc/linux/types.h

```diff
@@ -27,14 +27,18 @@ typedef	signed   __int64	__s64;
 typedef	unsigned __int64	__u64;
 
 
+#ifdef __REACTOS__
```

Comment:
these typedefs weren't in the original code?

Reply:
nope https://github.com/matt-wu/Ext3Fsd/blob/master/Ext3Fsd/include/linux/types.h#L31